bot_config.py

```diff
@@ -1,7 +1,16 @@
 #!/usr/bin/python
 
 # Discord bot token
-bot_token = ''
+try:
+	bot_token = open('token', 'r').read()
```

Comment:
This could be replaced with [python-dotenv](https://pypi.org/project/python-dotenv/), easier to maintain, hassle free :)